test/extended/util/openshift/clusterversionoperator/adminack.go

```diff
@@ -88,6 +112,7 @@ func (t *AdminAckTest) Test(ctx context.Context) {
 		if errMsg = setAdminGate(ctx, k, "true", t.Oc); len(errMsg) != 0 {
 			framework.Failf(errMsg)
 		}
+		exercisedGates[k] = exists
```

Comment:
exercisedGates could be nil here, right?

Reply:
Good catch; fixed with 4343107762 -> 79d0aa9ec5.